sys/include/net/gnrc/tcp.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015 Simon Brummer
+ * Copyright (C) 2017 Simon Brummer
```

Comment:
AFAIK, payment is not the (only) criteria, but that you did the implementation as part of a (supervised) thesis. Personally, I don't care and would not question your copyrights at all, I just try to avoid potential legal trouble in the future.